bcmc/src/main/java/com/adyen/checkout/bcmc/BcmcComponent.kt

```diff
@@ -46,7 +46,11 @@ class BcmcComponent internal constructor(
     ButtonComponent,
     ActionHandlingComponent by actionHandlingComponent {
 
-    override val delegate: ComponentDelegate get() = actionHandlingComponent.activeDelegate
+    override val delegate: ComponentDelegate
+        get() = when (val delegate = actionHandlingComponent.activeDelegate) {
```

Comment:
Why is this needed?

Reply:
This logic was used to get actual delegate before we started merging `viewFlow` using stateFlow. After merge with develop this logic is not needed anymore, so I'll remove it. Thanks for pointing